template/requirements.txt

```diff
@@ -0,0 +1 @@
+dwave-ocean-sdk==1.3.0
```

Comment:
Do you want to add any restrictions/guidelines on what dependencies are acceptable?

Reply:
I can't think of any hard restrictions at the moment. I will leave it to future reviewers to determine the appropriateness of a person's `requirements.txt`.